### ISSUE TITLE
Feature Request: ability to ask questionare during user creation. 

### ISSUE THREAD

**Issue by brianhenson** [Zuluru/Zuluru3#20]
Can we get the ability to ask questions from the questionnaires during user registration? This would be useful for capturing details about volunteers.

**Comment by Zuluru** [comment]
Do you mean the Questionnaires functionality that already exists? Or something different?

**Comment by brianhenson** [comment]
Yes basicly take the existing questionnaire and make it where you can marry
it to a role. Example would be if I created one for volunteer,  i could
then select that role in a menu and have it ask those questions
